pkg/chartverifier/helmactions.go

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2021 Red Hat
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package chartverifier
+
+import (
+	"bufio"
+	"io/ioutil"
+	"strings"
+
+	"helm.sh/helm/v3/pkg/action"
+	"helm.sh/helm/v3/pkg/chartutil"
+	kubefake "helm.sh/helm/v3/pkg/kube/fake"
+	"helm.sh/helm/v3/pkg/storage"
+	"helm.sh/helm/v3/pkg/storage/driver"
+	//"helm.sh/helm/v3/pkg/cli/values"
+
+	"github.com/redhat-certification/chart-verifier/pkg/helm/actions"
+)
+
+func getImages(chartUri string) ([]string, error) {
```

Comment:
there is the same function in `helm.go`, it seems.

Reply:
oops, have deleted helmactions.go